Cargo.toml

```diff
@@ -49,6 +49,10 @@ uint = { version = "0.9", default-features = false }
 url = { version = "2.3", default-features = false }
 uuid = { version = "1.1", default-features = false, features = [ "v4" ] }
 
+# Optional
+chrono = { version = "0.4", default-features = false, features = [ "std" ], optional = true }
```

Comment:
could we use `time` crate directly?

Reply:
Unfortunately no. `influxdb` only supports `chrono`.